mitiq/interface/conversions.py

```diff
@@ -70,10 +115,18 @@ def conversion_function(circ: Circuit) -> Circuit:
             return circ
 
     else:
-        raise UnsupportedCircuitError(
-            f"Circuit from module {package} is not supported.\n\n"
-            f"Circuit types supported by Mitiq are \n{SUPPORTED_PROGRAM_TYPES}"
-        )
+        for package_name, _ in register_from_dict:
+            if package_name in package:
+                input_circuit_type = package_name
+                conversion_function = register_from_dict[package_name, "from"]
+                break
+        else:
+            raise UnsupportedCircuitError(
+                f"Circuit from module {package} is not supported.\n\n"
+                f"Please define a converter with register_mitiq_converter(),"
+                f"\nor specify a supported Circuit type:"
```

Comment:
I believe this was a typo?
```suggestion
                f"\or specify a supported Circuit type:"
```

Reply:
This one's a line break. I'll add the space to clarify here and on similar cases nearby.

Reply:
I see, thanks!